.gitlab-ci.yml

```diff
@@ -112,6 +95,24 @@ Build and Install:
       - ${CI_PROJECT_DIR}/install
     expire_in: 1 day
 
+Build and Test:
+  stage: test
+  script:
+    - source ${CI_PROJECT_DIR}/install/thisREST.sh
```

Comment:
There is already a solution for this issue on #202.
On this other hand, I don't think that sourcing a compiled version of `rest` without the `testing` enabled is the way to go.

Reply:
> There is already a solution for this issue on https://github.com/rest-for-physics/framework/pull/202.

I see. It is fixed in 172ba772. No need to change pipeline now.

> On this other hand, I don't think that sourcing a compiled version of rest without the testing enabled is the way to go.

I don't understand. Does `tesing` make any difference to the compilation output? In principle it is compiling additional executables based on same library. So the PCM files can be reused.